src/task3.java

```diff
@@ -0,0 +1,152 @@
+        byte ЕкатеринаАндреевна = 30;
+        System.out.println("у Екатерины Андреевны " + ЕкатеринаАндреевна + " учеников");
+        short sheetsPurchasedTotal = 480;
+        System.out.println(sheetsPurchasedTotal);
+        int totalStudents = ЛюдмилаПавловна + АннаСергеевна + ЕкатеринаАндреевна;
+        System.out.println("всего учеников " + totalStudents);
+        int sheetForEachStudent = sheetsPurchasedTotal / totalStudents;
+        System.out.println("на каждого ученика расчитано " + sheetForEachStudent + " листов бумаги");
+
+        // Задача 4.
+        byte time2 = 16;
+        System.out.println("производит машина за 2 минуты = " + time2 + " штук бутылок");
+        int time20 = time2 * 20 / 2;
+        System.out.println("производит машина за 20 минут = " + time20 + " штук бутылок");
+        int time24 = time2 * 1440 / 2;
+        System.out.println("производит машина за 1440 минут = " + time24 + " штук бутылок");
+        int time4320 = time2 * 4320 / 2;
+        System.out.println("производит машина за 4320 минут = " + time4320 + " штуук бутылок");
+        int time43200 = time2 * 43200 / 2;
+        System.out.println("производит машина за 43 200 минут = " + time43200 + " штук бутылок");
+
+        // Задача 5.
+        byte brownPaint = 4;
+        byte whitePaint = 2;
+        int allPaints = brownPaint + whitePaint;
+        System.out.println("краски нужно на один класс = " + allPaints + " шт");
+        byte paint = 120;
+        byte allPaint = 6;
+        int classesAtSchool = paint / allPaint;
+        System.out.println("классов в школе = " + classesAtSchool + " шт");
+        int whitePaintOneClass = classesAtSchool * whitePaint;
+        System.out.println("В школе где 20 классов, нужно " + whitePaintOneClass + " банок белой краски");
+        int brownPaintOneClass = classesAtSchool * brownPaint;
+        System.out.println("В школе где 20 классов, нужно " + brownPaintOneClass + " банок коричневой краски");
+
+
+        // Задачат 6.
+        byte banans = 5;
+        byte weightOneBanana = 80;
+        int totalBanans = banans * weightOneBanana;
+        System.out.println(" в 5 бананах = " + totalBanans + "грамм");
+
+        int milk = 200;
+        byte in100Ml = 105;
+        int totalMilk = in100Ml * 2;
+        System.out.println("в 200 мл молока = " + totalMilk + " грамм");
+
+        byte iceCream = 2;
+        byte oneBriquette = 100;
+        int totalIceCream = oneBriquette * iceCream;
+        System.out.println("в двух брикетах = " + totalIceCream + " грамм");
+
+        byte rawEggs = 4;
+        byte gtamOneEggs = 70;
+        int gmTotalEggs = rawEggs * gtamOneEggs;
+        System.out.println("в четырех сырых яйцах = " + gmTotalEggs + " грамм");
+
+        int gmTotalProducts = totalBanans + totalMilk + totalIceCream + gmTotalEggs;
+        System.out.println("во всех продуктах = " + gmTotalProducts + " граммов");
+
+        float kg = gmTotalProducts / 1000;
```

Comment:
деление целочисленное, и ответ получается неправильный, потому что 5 / 2 - это 2, а не 2.5. Чтобы было 2.5, надо делать 5.0 на 2 или 5 на 2.0
```suggestion
        float kg = gmTotalProducts / 1000F;
```

Reply:
Аааа.. понял, спасибо